### PR TITLE
Issue #3: Added audit log used message

### PR DESCRIPTION
**Pull request by hnnesv**
@rcrichton would mind reviewing this? thanks

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -231,3 +256,23 @@ exports.appActivityAudit = function(isStart, sysname, hostname, username) {
   var audit = new AuditMessage(eIdent, [sysParticipant, userParticipant], null, [sourceIdent]);
   return audit.toXML();
 };
+
+exports.auditLogUsedAudit = function(outcome, sysname, hostname, username, userRole, userRoleCode, auditLogURI, objQuery, objDetails) {
+  var eventID = new Code(110101, 'Audit Log Used', 'DCM');
+  var eIdent = new EventIdentification(exports.EVENT_ACTION_READ, new Date(), outcome, eventID, null);
+
+  var sysRoleCode = new Code(110150, 'Application', 'DCM');
+  var sysParticipant = new ActiveParticipant(sysname, '', true, hostname, exports.NET_AP_TYPE_DNS, [sysRoleCode]);
```
Comment (rcrichton) [inline index.js]: I think the `userIsRequestor` boolean when using `sysname` should be false here. I noticed this while doing mine. I've changed the others in my branch.

**Comment by rcrichton** [comment]
This looks good to me. Just one comment above.

I wish there was some way we could validate these more thoroughly, it's pretty hard to ensure these are correct against the standard over and above the schema. I haven't been able to find the connectathon schematrons yet, those would be ideal.

File: index.js [hunk 2]
```diff
@@ -231,3 +256,23 @@ exports.appActivityAudit = function(isStart, sysname, hostname, username) {
   var audit = new AuditMessage(eIdent, [sysParticipant, userParticipant], null, [sourceIdent]);
   return audit.toXML();
 };
+
+exports.auditLogUsedAudit = function(outcome, sysname, hostname, username, userRole, userRoleCode, auditLogURI, objQuery, objDetails) {
```
Comment (rcrichton) [inline index.js]: Afterthought: perhaps you could add a JSDoc comment here. I'm doing that on my branch. It would help to have these functions documented.

**Comment by hnnesv** [comment]
Yeah :/ and it's difficult to find anything from IHE about what they want for the "Not specialized" fields

File: index.js (hunk 2 above)
Comment (hnnesv) [inline index.js]: Ok cool - will add

**Comment by hnnesv** [comment]
Gazelle validation passes but npm test fails. Needs #5 in order to complete

**Comment by hnnesv** [comment]
@rcrichton mind taking another look at this? Gazelle's happy with the message now too.

File: index.js [hunk 3]
```diff
@@ -239,6 +264,39 @@ exports.appActivityAudit = function(isStart, sysname, hostname, username) {
 };
 
 /**
+ * Generates a 'Audit Log Used' audit message in XML format
+ * @param  {Number} outcome       the desired outcome, for authentication failure use atna.OUTCOME_MINOR_FAILURE.
```
Comment (rcrichton) [inline index.js]: I think there is a copy-paste problem with this description, it shouldn't reference authentication failure.

**Comment by rcrichton** [comment]
@devcritter just one comment above, otherwise :+1:

**Pull request merged**